packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -164,32 +173,57 @@ export class SOQLEditorInstance {
         this.updateTextDocument(this.document, soql);
         break;
       }
+      case MessageType.UI_TELEMETRY: {
+        const { unsupported, errors } = e.payload as TelemetryModelJson;
+        if (errors && errors.length) {
+          trackError('syntax_error', JSON.stringify(e.payload)).catch(
+            console.error
+          );
+        }
+        if (unsupported && unsupported.length) {
+          // no need to duplicate.  unsupported and errors often both present
+          if (!errors || !errors.length) {
+            trackError('syntax_unsupported', JSON.stringify(e.payload)).catch(
+              console.error
+            );
+          }
+          channelService.appendLine(
+            `This syntax is not yet supported: ${unsupported.join(', ')}`
```

Comment:
@ekapner - This message would appear in the channel output.  it mirrors what is displayed in the UI.  Except this shows the unsupported syntax statements like from this example.

`This syntax is not yet supported: WHERE`

<img width="972" alt="Screen Shot 2020-12-07 at 1 26 27 PM" src="https://user-images.githubusercontent.com/599418/101409703-7d683600-389b-11eb-99e0-391df7201b62.png">

Reply:
@dehru - is there a workaround?

Reply:
`The syntax is not yet supported in SOQL Builder.  Use a text editor.`